### PR TITLE
Added EBS Volumes Tagging

### PR DESCRIPTION
**Pull request by ghost**
@e0d , can you please review the changes. Thanks

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @arbab-edx! I've created [OSPR-301](https://openedx.atlassian.net/browse/OSPR-301) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams. 

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ('this must be merged by XX date', and why that is)
- partner information ('this is a course on edx.org')
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the Github pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

We can't start reviewing your pull request until you've submitted a [signed contributor agreement](http://code.edx.org/individual-contributor-agreement.pdf) or indicated your institutional affiliation and added yourself to the [AUTHORS](https://github.com/edx/configuration/blob/master/AUTHORS) file. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst) file for more information.

**Comment by feanil** [comment]
@singingwolfboy @arbab-edx is actually a contractor that works with the devops team so I believe he shouldn't need an individual CLA is that right?  What determines whether or not the webhook perceives him as an outside contributor?

File: playbooks/roles/supervisor/files/pre_supervisor_checks.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def edp_for_instance(instance_id):
     parser = argparse.ArgumentParser(
         description="Enable all services that are in the services tag of this ec2 instance.")
     parser.add_argument("-a","--available",
-        help="The location of the available services.")
+                        help="The location of the available services.")
```
Comment (feanil) [inline playbooks/roles/supervisor/files/pre_supervisor_checks.py]: Why did the spacing for this get changed?

File: playbooks/roles/supervisor/files/pre_supervisor_checks.py [hunk 2]
```diff
@@ -114,7 +114,21 @@ def edp_for_instance(instance_id):
             play=play,
             instance_id=instance_id)
     except:
-        print("Failed to get EDP for {}".format(instance_id))
+        print("Failed to get EDP for {}".format(instance=instance_id))
+
+    #get the hostname of the sandbox
+    hostname = socket.gethostname()
```
Comment (feanil) [inline playbooks/roles/supervisor/files/pre_supervisor_checks.py:121]: socket module was not imported.

**Comment by singingwolfboy** [comment]
@feanil: botbro uses [people.yaml](https://github.com/edx/repo-tools/blob/master/people.yaml) to determine who is who. Looks like @arbab-edx doesn't have an entry in that file. If we don't want botbro to make OSPR issues for his pull requests, we'll need to get his name, email, and institutional affiliation (plus JIRA username, if he has one) so that we can add him to that file.

**Comment by ghost** [comment]
@e0d , please check the modified pre_supervisor_checks.py, Thanks

File: playbooks/roles/supervisor/files/pre_supervisor_checks.py (hunk 1 above)
Comment (ghost) [inline playbooks/roles/supervisor/files/pre_supervisor_checks.py]: @feanil, sorry add the wrong file, please check the updated one. Thanks

**Comment by feanil** [comment]
looks good to me.

**Comment by e0d** [comment]
:+1:

**Pull request merged**